Update Power BI Azure Maps visual doc (#126237)
Azure/azure-content: articles/azure-maps/power-bi-visual-get-started.md

| Setting     | Description  |
|-------------|--------------|
| Style       | The style of the map. The dropdown list contains [blank and blank accessible], [grayscale dark], [grayscale light], [high contrast dark], [high contrast light], [night], [road], [road shaded relief], [satellite] and [satellite road labels]. |

> | Labels | A toggle switch that allows you to show or hide map labels. For more information, see list item number five in the previous section. |

Reviewer: ```suggestion
| Labels | A toggle switch that allows you to show or hide map labels. For more information, see item number five in the previous section titled [Use the Azure Maps Power BI visual](#use-the-azure-maps-power-bi-visual). |
```

> | Labels | A toggle switch that enables you to either show or hide map labels. For more information, see list item number five in the previous section. |

Reviewer: ```suggestion
| Labels | A toggle switch that allows you to show or hide map labels. For more information, see list item number five in the previous section. |
```

> | Labels | A toggle switch that enables you to either show or hide map labels. For more information, see list item number five in the previous section. |

Reviewer: What/where is list item #5? I'm not sure what this is refering to?

Author: It refers to the section at line 88.

| Country borders | A toggle switch that controls the visibility of country borders. |
| State or province borders | A toggle switch that controls the visibility of the borders for the first-level administrative divisions, such as state or province borders. |
| County borders | A toggle switch that controls the visibility of the borders for the second-level administrative divisions, such as county borders. |
| Buildings | A toggle switch that controls the visibility of building footprints |
| Road details | A toggle switch that adjusts the level of detail for visible roads. Turning this off will reduce the number of roads shown on the map. |

> The availibity of the above toggles depends on the style you select. Here is a table that shows the supported options for different styles:

Reviewer: ```suggestion
The availability of the above toggles depends on the selected style. Below is a table showing the supported options for different styles:
```

| Map styles | Labels | Country borders | State or province borders | County borders | Buildings | Road details |
|-|-|-|-|-|-|-|
| [blank and blank accessible] | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
| [grayscale dark] | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ |
| [grayscale light] | ✅ | ✅ | ✅ | ❌ | ✅ | ✅ |
| [high contrast dark] | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| [high contrast light] | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| [night] | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| [road] | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| [road shaded relief] | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
| [satellite] | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
| [satellite road labels] | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |

> If an option is not supported by the current selected style, it simply has no effect until you select another style that supports it.

Reviewer: ```suggestion
If an option is not supported by the currently selected style, it will have no effect until you choose a style that supports it.
```
